site/src/pages/signup.page.tsx

```diff
@@ -240,12 +246,11 @@ const SignupPage: NextPage = () => {
               [
                 {
                   icon: (
-                    <Icon
+                    <ArrowUpCircleIcon
```

Comment:
**Potentially blocking:** did you deliberately use different icons?
Previously:
<img width="365" alt="image" src="https://user-images.githubusercontent.com/42802102/157713195-0b6875f5-b4a0-4475-a3ac-0c33f3d5b089.png">
Now:
<img width="371" alt="image" src="https://user-images.githubusercontent.com/42802102/157713226-e3c29da9-7ecf-472c-b26c-71c1c13d6d72.png">

Reply:
Yes, custom icons were created to replace those icons [internal link](https://hashintel.slack.com/archives/C02NW18QGUR/p1646821021066609?thread_ts=1637770288.003200&cid=C02NW18QGUR)